flink-core/src/main/java/org/apache/flink/api/common/state/State.java

```diff
@@ -23,7 +23,7 @@
 /**
  * Interface that different types of partitioned state must implement.
  *
- * <p>The state is only accessible by functions applied on a KeyedDataStream. The key is
+ * <p>The state is only accessible by functions applied on a KeyedStream. The key is
```

Comment:
Could you also change these to be actual Javadoc links?
i.e. `{@link KeyedStream}`.